### ISSUE TITLE
Crash when updating data from overpass

### ISSUE THREAD

**Issue by RedAuburn** [a-b-street/abstreet#1094]
`thread 'main' panicked at 'Couldn't find the cli executable. Is it built?', map_gui/src/tools/mod.rs:399:5`

[output.txt](https://github.com/a-b-street/abstreet/files/12182930/output.txt)

on fedora 38

**Comment by dabreegster** [comment]
Thanks for reporting, I can confirm this is broken in the 0.3.45 release and probably 0.3.44 too -- e615504a87d462a6c7bff657fac294b481cc3d66 broke it by organizing the files in the release .zip. I'll fix and push a new release when I have time, no guarantees. 0.3.43 may work in the meantime

**Comment by dabreegster** [comment]
Just a note to myself: this fix uncovered another problem. One of the log lines for importing causes text rendering to break at `widgetry/src/text.rs:518`. Reproduce with this GJ import:

```
{
  "type": "FeatureCollection",
  "features": [
    {
      "type": "Feature",
      "properties": {},
      "geometry": {
        "coordinates": [
          [
            [
              -72.94401338580232,
              41.34358255224146
            ],
            [
              -72.94401338580232,
              41.29666713691714
            ],
            [
              -72.8936131373086,
              41.29666713691714
            ],
            [
              -72.8936131373086,
              41.34358255224146
            ],
            [
              -72.94401338580232,
              41.34358255224146
            ]
          ]
        ],
        "type": "Polygon"
      }
    }
  ]
}
```

Fix that before doing another release

**Comment by dabreegster** [comment]
This'll be fixed properly in 0.3.47, coming later today. Thanks for the patience!
